### PR TITLE
remove baremetalhost crd

### PR DESCRIPTION
**Pull request by dhellmann**
The CRD is installed as part of launching the MAO, so we do not need to
install it here.

### CODE REVIEW

**Comment by dhellmann** [comment]
See https://bugzilla.redhat.com/show_bug.cgi?id=1748452 and https://github.com/openshift/machine-api-operator/issues/397

**Comment by dhellmann** [comment]
The machine-api-operator now registers the CRD correctly, so we should be able to land this.

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1235/

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1237/

File: bootstrap_bootkube_log.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$(dirname $0)/show_bootstrap_log.sh bootkube.service
```
Comment (stbenjam) [inline bootstrap_bootkube_log.sh]: Was this supposed to be included here?

Comment (dhellmann) [reply bootstrap_bootkube_log.sh]: Yeah, I needed that to debug the other change so I just left it in this PR. It's a separate commit for tracking later.

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1245/

**Comment by hardys** [comment]
> Maybe we should up the CI timeout?

We could try but I suspect we'll just hit the installer timeout instead, hopefully the slow-download issue is a temporary thing

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1249/

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1253/

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1255/

File: bootstrap_bootkube_log.sh (hunk 1 above)
Comment (zaneb) [reply bootstrap_bootkube_log.sh:3]: It appears not unidentical to this other one though: https://github.com/openshift-metal3/dev-scripts/pull/763/files#diff-19647512eab29fd4eda6d2f9dc694ee7

**Pull request merged**